adam_core/utils/tests/test_indexable.py

```diff
@@ -230,6 +245,131 @@ def test__convert_to_array():
     np.testing.assert_equal(Indexable._convert_to_array(value), desired)
 
 
+def test_Indexable__eq__():
```

Comment:
It's not a very big deal, but personally I'd break this into many many littler tests. I think it's easier to read, and if you have failures, you'll get _all_ the failures instead of the first one only.

Reply:
Done!